### PR TITLE
VZ-5055 Remove Integrtion Tests from Pipeline

### PR DESCRIPTION
**Pull request by bryantidd**
# Description

Removed the Stage Integration Tests from the pipeline
Fixes VZ-5055

# Checklist 

As the author of this PR, I have:

- [ ] Checked that I included or updated copyright and license notices in all files that I altered
- [ ] Added or updated unit tests for any new functions I added
- [ ] Added or updated integration tests if appropriate
- [ ] Added or updated acceptance tests if appropriate

Code reviewer, please confirm this PR:

- [ ] Addressed the requirement and meets the acceptance criteria
- [ ] Does not introduce unrelated or spurious changes
- [ ] Does not introduce any unapproved dependency
- [ ] Makes sense and it easy to understand, and/or difficult areas of code are clearly documented so that they can be understood

### CODE REVIEW

**Review by agv426 — APPROVED** [review]

**Review by stevshap — DISMISSED** [review]

File: Jenkinsfile [hunk 1]
```diff
@@ -389,24 +389,6 @@ pipeline {
             }
         }
 
-        stage('Integration Tests') {
-            when { not { buildingTag() } }
-            steps {
-                integrationTests("${DOCKER_IMAGE_TAG}")
```
Comment (bobdonat) [inline Jenkinsfile:395]: shouldn't the integrationTests def be removed too since it not being used now

Comment (bobdonat) [reply Jenkinsfile:395]: and the SIMULATE_FAILURE parameter can be removed since it is only referenced by integration tests

Comment (agv426) [reply Jenkinsfile:395]: missed that, we should move that to fail from a different stage rather than remove the param

**Review by agv426 — APPROVED** [review]

File: Jenkinsfile (hunk 1 above)
Comment (bryantidd) [reply Jenkinsfile:395]: SIMULATE_FAILURE will move to a separate task.

**Pull request merged**